vyper/venom/parser.py

```diff
@@ -11,13 +13,14 @@
 from vyper.venom.context import IRContext
 from vyper.venom.function import IRFunction
 
-VENOM_PARSER = Lark(
-    """
+VENOM_GRAMMAR = """
     %import common.CNAME
     %import common.DIGIT
     %import common.LETTER
     %import common.WS
     %import common.INT
+    %import common.SIGNED_INT
+    %import common.ESCAPED_STRING
```

Comment:
Just wondering if it would be better to just change up some of the labels instead of parsing escaped string (I dont know how many cases are there I stumbled so far on case with the space which could be replaced by underscore)?

Reply:
i could do it both ways honestly, but this seemed nicer for debugging (you can imagine the frontend generating a sentence or description of where it came from for a label)